javascripts/discourse/connectors/bread-crumbs-right/unanswered-filter-connector.hbs

```diff
@@ -1 +1,3 @@
-<UnansweredFilter />
\ No newline at end of file
+{{#if (eq (theme-setting "filter_mode") "dropdown")}}
```

Comment:
This is quite inefficient as the component is rendered even when the theme setting is not set to `dropdown`.  Let's move this to use the `api.renderInOutlet` API coupled with the `refresh: true` suggestion below so that we only render the component when required.

Reply:
The component will still be rendered but just that the template will render nothing if the condition is not true.